deid/bene_id_mapping.sql

```diff
@@ -13,105 +14,79 @@ select
 from (
   select /*+ PARALLEL(HHA_OCCURRNCE_CODES,12) */ distinct bene_id from HHA_OCCURRNCE_CODES union
   select /*+ PARALLEL(PDE,12) */ distinct bene_id from PDE union
-  select /*+ PARALLEL(BCARRIER_CLAIMS,12) */ distinct bene_id from BCARRIER_CLAIMS union
+  select /*+ PARALLEL(BCARRIER_CLAIMS_K,12) */ distinct bene_id from BCARRIER_CLAIMS_K union

```

Comment:
This wouldn't work with 2011-2013 data, would it? I guess that's reasonable. I wonder about a separate branch.... Maybe we should tag the code we used for 2011-2013.

Reply:
tag has been created. https://github.com/kumc-bmi/grouse/releases/tag/2011-2013_CMS_DEID